test/index.js

```diff
@@ -877,40 +877,48 @@ describe('normalizr', function () {
     });
   });
 
-  it('can normalize nested entities', function () {
```

Comment:
Oups, my mistake, thought I was copy pasting and apparently rewrote the test instead, let me fix that!